go/vt/vttablet/endtoend/framework/client.go

```diff
@@ -415,9 +415,11 @@ func (client *QueryClient) UpdateContext(ctx context.Context) {
 }
 
 func (client *QueryClient) GetSchema(tableType querypb.SchemaTableType, tableNames ...string) (map[string]string, error) {
-	schemaDef := map[string]string{}
+	schemaDef := make(map[string]string)
 	err := client.server.GetSchema(client.ctx, client.target, tableType, tableNames, func(schemaRes *querypb.GetSchemaResponse) error {
-		schemaDef = schemaRes.TableDefinition
+		for tableName, schemaDefinition := range schemaRes.TableDefinition {
+			schemaDef[tableName] = schemaDefinition
+		}
```

Comment:
This was another bug I found! We shouldn't just be overriding the map we get back, but we should update it.